### DISCUSSION TITLE
csstype error when build to production

### DISCUSSION THREAD

**Discussion by bill-lee-aics** [element-plus/element-plus#3545] [Q&A]
version of packages:
element-plus: 1.1.0-beta.12

There is no error in dev enviroment.
But when it comes to production, there is lots of error message like
```
node_modules/.pnpm/element-plus@1.1.0-beta.12_vue@3.2.6/node_modules/element-plus/lib/components/popper/src/use-popper/index.d.ts:797:42 
- error TS2694: Namespace '"/Users/xxlee/care-management-frontend/node_modules/.pnpm/csstype@3.0.8/node_modules/csstype/index"' has no exported member 'VectorEffectProperty'.
```
I installed `vue-jss` in this project, which has finally a `csstype` v3 dependency. It seems like it import `csstype` v3 here and result in these error.
I also tried to build with `npm` instead of `pnpm` but still came into same result.
Is there any solution to this?

BTW, there is also some type error like
```
node_modules/.pnpm/element-plus@1.1.0-beta.12_vue@3.2.6/node_modules/element-plus/lib/components/time-picker/src/common/picker.vue.d.ts:2:24 - error TS2307: Cannot find module 'element-plus/es/popper' or its corresponding type declarations.

import { Effect } from 'element-plus/es/popper';
```

Thanks in advance.

**Comment by jw-foss** [comment]
cc @sxzz

**Reply by jw-foss** [reply]
Also @bill-lee-aics can you raise an issue instead of discussion here so we can prioritize and evaluate your issue? Thanks!

**Comment by sxzz** [comment]
Could you describe the steps to reproduce this problem?
